cumulusci/tasks/salesforce/ListCommunityTemplates.py

```diff
@@ -0,0 +1,19 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+
+
+class ListCommunityTemplates(BaseSalesforceApiTask):
+    api_version = "46.0"
+    task_docs = """
+    Lists Salesforce Community templates available for the current org via the Connect API.
+    """
+    task_options = {}
+
+    def _run_task(self):
+        community_template_list = self.sf.restful("connect/communities/templates")[
+            "templates"
+        ]
+
+        self.logger.info(
+            "Community Templates available to the current org:%s",
```

Comment:
```suggestion
            "Community Templates available to the current org:\n%s",
```

so that the first template appears on its own line.